FruitFruit/FruitFruit/Views/HomeStoryboard/FruitStatusLabel.swift

```diff
@@ -93,5 +93,6 @@ class FruitStatusLabel: UIView {
     
     func setLabelImage(from text: String) {
         statusImage.image = UIImage(named: text)
+        statusImage.frame = CGRect(x:0, y:0, width: 130, height: 130)
```

Comment:
```suggestion
        statusImage.frame.sizea = CGSize(width: 130, height: 130)
```

origin에 영향이 있을 수 있으니, 조정이 필요한 Size만 설정할 수 있습니다 ~

Reply:
오 감사합니다!